nench.sh

```diff
@@ -185,12 +186,15 @@ uname -s -r -m
 printf '\n'
 
 printf 'Disks:\n'
-if command_exists lsblk
+if command_exists lsblk && [ -n "$(lsblk)" ]
```

Comment:
Have you ever seen `lsblk` produce actually empty output?
According to its own manpage, it should always produce output (at least the header row), so this check wouldn't actually help here…

Reply:
On ovz i don't see any output

Reply:
it even gives exit code `1` if you prefer that